### DISCUSSION TITLE
Can Mytourbook prefill the starting point of an activity?

### DISCUSSION THREAD

**Discussion by froggyranita2** [mytourbook/mytourbook#926] [Q&A] [answered]
Hello, can Mytourbook during the import prefill the starting/end point of an activity based on the GPS trace or from the entered location of a previous activity as it is the case with SportTracks?
I searched in the forum but i could not find an answer.
Thanks
Pascal

**Comment by wolfgang-ch** [comment, marked as answer]
This is already requested in https://github.com/mytourbook/mytourbook/issues/878
